curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/65687d2f8c7ee27b0446cef3.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 65687d2f8c7ee27b0446cef3
+title: Step 19
+challengeType: 20
+dashedName: step-19
+---
+
+# --description--
+
+The code to execute at each iteration — placed after the `:` — constitutes the body of the loop. This code must be indented — typically 4 spaces — from the loop level. This indented level is a code block.
```

Comment:
```suggestion
The code to execute at each iteration — placed after the `:` — constitutes the body of the loop. This code must be indented. In Python, it is recommended to use 4 spaces per indentation level. This indented level is a code block.
```

Reply:
I would suggest emphasizing that the 4 spaces guideline applies specifically for Python.